### PR TITLE
Add overview.md

### PR DESCRIPTION
**Pull request by EasyRhinoMSFT**

### CODE REVIEW

File: src/Sarif.Viewer.VisualStudio/overview.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The Microsoft SARIF Viewer extension for Visual Studio provides a convenient UI for analyzing static analysis log files and taking action on the items they contain. The Viewer intgrates with the Visual Studio environment, displaying a list of analysis results in the Error List and result details in a dockable tool window. Embedded target files can be opened right in the text editor with the error line highlighted. If the target file isn't embeeded in the log file, the Viewer will prompt you to open a local copy.
```
Comment (jeff-bacon) [inline src/Sarif.Viewer.VisualStudio/overview.md]: >Viewer [](start = 169, length = 6)

Should be lower case

Comment (jeff-bacon) [inline src/Sarif.Viewer.VisualStudio/overview.md]: >intgrates [](start = 176, length = 9)

Spelled incorrectly

Comment (jeff-bacon) [inline src/Sarif.Viewer.VisualStudio/overview.md]: >Error List [](start = 267, length = 10)

Should be lower case

Comment (jeff-bacon) [inline src/Sarif.Viewer.VisualStudio/overview.md]: >embeeded [](start = 443, length = 8)

Incorrect spelling

Comment (jeff-bacon) [inline src/Sarif.Viewer.VisualStudio/overview.md]: >Viewer [](start = 473, length = 6)

should be lower case

Comment (EasyRhinoMSFT) [reply src/Sarif.Viewer.VisualStudio/overview.md]: I'm capitalizing it as it's part of the full name, "SARIF Viewer."

Comment (EasyRhinoMSFT) [reply src/Sarif.Viewer.VisualStudio/overview.md]: This is the formal name of that listview instance. It is capitalized this way in VS documentation.

Comment (EasyRhinoMSFT) [reply src/Sarif.Viewer.VisualStudio/overview.md]: Added "SARIF"

Comment (EasyRhinoMSFT) [reply src/Sarif.Viewer.VisualStudio/overview.md]: Added "SARIF"

**Pull request merged**